src/main/webapp/i18n/en/modelingExercise.json

```diff
@@ -13,6 +13,7 @@
             "detail": {
                 "title": "Modeling Exercise"
             },
+            "automaticAssessmentEnabled": "Enable automatic Assessment",
```

Comment:
```suggestion
            "automaticAssessmentEnabled": "Enable Automatic Assessment",
```

Reply:
If that is alright, I will do this in a followup (which is already set up #7743) to get this one merged